tests/testthat/test-npn-observations.R

```diff
@@ -122,6 +136,20 @@ test_that("phenometrics downloads work", {
 
   expect_gt(num_mag_custom, num_mag_default)
 
+  vcr::use_cassette("npn_download_no_records", {
+    expect_message(
+      dl <- npn_download_individual_phenometrics(
+        request_source = 'erinz',
+        years = 2009,
+        species_ids = 1612
+      ),
+      "No records in 2009"
+    )
+  })
+
+  #remove one of these depending on which you decide to return on this error
+  expect_equal(dl, dplyr::tibble())
+  expect_null(dl)
```

Comment:
`dl` does get created even though it is created inside of `expect_message()` and you can check that it is either an empty tibble or it is NULL (you choose and delete whichever test doesn't apply)

Reply:
thanks @Aariq - this is all super helpful.  I just pulled and reviewed the changes. I think you're probably right that an empty tibble rather than a NULL object would be better.  I deleted `expect_null(dl)` from the test script and changed the associated line in npn_data_download.R. I assume I should push these changes?  Do I need to do anything else on my end? Thanks so much for walking me through this!